lib/ExponentialBackoff.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+var ExponentialBackoff = function(minimum_backoff, exponent, starting_backoff) {
+    if (minimum_backoff === undefined) {
+        minimum_backoff = 50;
+    }
+
+    if (exponent === undefined) {
+        exponent = 2.0;
+    }
+
+    if (starting_backoff === undefined) {
+        starting_backoff = minimum_backoff;
+    }
+
+    var backoff = starting_backoff;
+
+    return {
+        getBackoff: function() {
```

Comment:
Actually that's a perfect example, and thanks for the link. My copy of Javascript: The Good Parts is somewhere in the middle of the country right now, or I'd refer to it. 

Is it more common in the node world for the module.exports to be the constructor function, or a constructed object? Seems like [here](https://github.com/missinglink/ciao/blob/master/lib/Settings.coffee) for example, requiring the module gives you a constructor that can be used multiple times, whereas with your stats module, if you wanted multiple instances you would have to call require multiple times.